recipes/configs/eleuther_eval.yaml

```diff
@@ -0,0 +1,30 @@
+# Config for EleutherEvalRecipe in eleuther_eval.py
+#
+# To launch, run the following command from root torchtune directory:
+#    tune eleuther_eval --config llama2_eleuther_eval tasks=["truthfulqa_mc2", "hellaswag"]
+
+# Model Arguments
+model:
+  _component_: torchtune.models.llama2.llama2_7b
+
+checkpointer:
+  _component_: torchtune.utils.FullModelTorchTuneCheckpointer
+  checkpoint_dir: /tmp/llama/
+  checkpoint_files: [finetuned_model.pt]
```

Comment:
Actually the TorchTune checkpointer outputs a model with a specific name format. Maybe we should update the config to that name so folks can use OOTB

Reply:
I commented this somewhere else, but the specific name format requires an epoch number so there's no way to know exactly what the checkpoint file will be. I'm fine changing this name to *resemble* the outputted file, but it will not match OOTB.